app/scenes/Settings/components/ApiKeyListItem.tsx

```diff
@@ -17,6 +19,12 @@ const ApiKeyListItem = ({ apiKey }: Props) => {
   const { t } = useTranslation();
   const [linkCopied, setLinkCopied] = React.useState<boolean>(false);
 
+  const subtitle = apiKey.lastUsedAt ? (
```

Comment:
For the existing keys, we do not have info on its last use; so we render a `null` subtitle.